src/main/java/nextstep/subway/domain/Sections.java

```diff
@@ -28,65 +31,90 @@ public void add(Section section) {
             addSectionInMiddle(section);
         }
     }
+    public List<Station> getOrderedStations(){
+        Station currentStation = getFirstUpStation();
+        List<Station> orderedStations = new ArrayList<>();
+        orderedStations.add(currentStation);
+        while (orderedStations.size()-1 < sections.size()){
+            currentStation = matchingSection(currentStation).getDownStation();
+            orderedStations.add(currentStation);
+        }
+        return orderedStations;
+    }
+
+    public Optional<Section> getSectionById(Long sectionId){
+        return sections.stream().filter(section -> section.getId().equals(sectionId)).findFirst();
+    }
+
+    public List<Section> getAll() {
+        return sections;
+    }
+
+
+    public void deleteStation(Station station) throws NotFoundException {
```

Comment:
```suggestion
    }

    public void deleteStation(Station station) throws NotFoundException {
```

자동 포멧팅 설정해놓으면 편해요~
개행처리가 불규칙해보여요